### DISCUSSION TITLE
Microsoft edge web view2 runtime upgrading using winget

### DISCUSSION THREAD

**Discussion by DanialSafaei** [microsoft/winget-pkgs#44812] [General] [answered]
Installer failed with exit code: 2147748098
![Screenshot 2022-02-05 132601](https://user-images.githubusercontent.com/82803206/152636971-d54eae44-ffed-40ad-8b61-d28c5fbf91b7.png)

**Comment by OfficialEsco** [comment, marked as answer]
You need to run Powershell as Administrator
The Manifest does have the correct field `ElevationRequirement: elevationRequired` but [winget-cli](https://github.com/microsoft/winget-cli) does currently not respect this field.
